addOns/ascanrulesBeta/src/main/java/org/zaproxy/zap/extension/ascanrulesBeta/ProxyDisclosureScanRule.java

```diff
@@ -752,7 +752,7 @@ public void scan() {
                                 Constant.messages.getString(
                                         MESSAGE_PREFIX + "desc",
                                         step2numberOfNodes - 1 + silentProxySet.size()))
-                        .setAttack(getAttack())
+                        .setAttack(Constant.messages.getString(MESSAGE_PREFIX + "attack"))
```

Comment:
This doesnt actually look right, I wouldnt expect attack strings to be i18ned. I realise this is how it was before...

Reply:
Good point, I'll dig further into what it's doing there.

Reply:
Do you agree that this text doesn't really make any sense or add any value? If so I'll just remove it.
`ascanbeta.proxydisclosure.attack = TRACE, OPTIONS methods with 'Max-Forwards' header. TRACK method.`

I had considered adding it to the Other Info details but I don't really even understand what this is supposed to convey.